core/controllers/feedback.py

```diff
@@ -215,5 +220,33 @@ def get(self, exploration_id):
 class UnsentFeedbackEmailHandler(base.BaseHandler):
     """Handler task of sending emails of feedback messages.
     This is yet to be implemented."""
+
     def post(self):
-        pass
+        payload = json.loads(self.request.body)
+        user_id = payload['user_id']
+        references = feedback_services.get_feedback_message_references(user_id)
+        transaction_services.run_in_transaction(
+            feedback_services.update_feedback_email_retries, user_id)
+
+        messages = {}
+        for reference in references:
+            message = feedback_services.get_message(
+                reference.exploration_id, reference.thread_id,
+                reference.message_id)
+
+            exploration = exp_services.get_exploration_by_id(
+                reference.exploration_id)
+
+            message_text = message.text
+            if len(message_text) > 200:
+                message_text = message_text[:200] + '...'
+
+            if exploration.title in messages:
```

Comment:
Yes, I think it's still better to do it properly. In the future I can envision us changing the format and organization of the email, and we don't want to have to deal with a transitory state where we have to make non-trivial updates to the model structure.